src/components/CHeader.vue

```diff
@@ -15,6 +47,37 @@
 <script>
 export default {
   name: 'CHeader',
-  components: {}
+  components: {},
+  methods: {
+    onUpdateMessages() {
+      this.$emit('onUpdateMessages');
+    },
+    onUpdateRefreshTime() {
+      this.$emit('onUpdateRefreshTime', this.refreshTimeSelected);
+    }
+  },
+  data() {
+    return {
+      refreshTimeSelected: 10,
+      updateTimes: [
+        { text: 'off', value: -1 },
```

Comment:
i would avoid to use a negative value, seems kind of a hack to setInterval. I would prefer that value=null and before calling setInterval you check if (refreshTime). also rename refreshTime to resfreshInterval. and set the default to 30s.